.vale/styles/config/vocabularies/docs/accept.txt

```diff
@@ -1,28 +1,32 @@
 # Please keep this list sorted (alphabetically, case-insensitive)
-Alertmanager
+[Aa][Ww][Ss]
 [Aa]gent
+[cC]loud
+[Cc]rossplane
+[Dd][Nn][Ss]
+[Dd]atasource[s]?
+[Dd]ex
+[Ii][Aa][Mm]
```

Comment:
It feels weird to have square bracelets variants first and not on the alphabetical list though